### PR TITLE
Miscellaneous CI job cleanups

### PR DESCRIPTION
**Pull request by nixpanic**
PR #1489 introduced new jobs using a JJB template, but it fails to deploy as the referencing of the template seems incorrect.

There is also a `sandbox` option in all jobs, but this option is not recognized and causes warnings in the Jenkins WebUI.

### CODE REVIEW

**Comment by nixpanic** [comment]
**ci/centos/jjb-validate** fails, it seems to include the jobs that are part of the current deployment. Not sure it runs against the modified jobs by PRs :thinking:

**Comment by nixpanic** [comment]
> **ci/centos/jjb-validate** fails, it seems to include the jobs that are part of the current deployment. Not sure it runs against the modified jobs by PRs thinking

Manually changed the jjb-validate job to what has been added to this PR. :crossed_fingers:

**Comment by Madhu-1** [comment]
Still validation is failing

File: jobs/ci-job-validation.yaml
Comment (Yuggupta27) [inline jobs/ci-job-validation.yaml:5]: Will any new scripts now require an extra admin approval after getting merged?

Comment (nixpanic) [reply jobs/ci-job-validation.yaml:5]: no, that is not changing

**Comment by nixpanic** [comment]
> Still validation is failing

Yes, I found a 2nd spot where the wrong GIT_REF is used. That has been corrected now too

**Comment by nixpanic** [comment]
There is a change needed in `deploy/checkout.sh` which is part of the container image that is built from the ci/centos branch. **ci/centos/jjb-validate** will keep failing until the container image is rebuilt with the change from this PR.

**Review by Madhu-1 — APPROVED** [review]

**Review by Yuggupta27 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@nixpanic can we merge this PR?

**Pull request merged**
